Source/Core/VideoBackends/D3D/PixelShaderCache.cpp

```diff
@@ -150,12 +150,12 @@ const char depth_matrix_program[] = {
 
```

Comment:
Isn't using (1.0 - texcol.x) easier than using 255 - XXX all over the place here?